### PR TITLE
Vocabulary instructions ready for review

### PR DESCRIPTION
**Pull request by dxwell**
@KellyVance please review the vocabulary instructions.
Still 2 outstanding items - a link to the create vocabs Excel file and John to add the instructions on registering vocabs at linkedata.gov.au

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,27 +1,64 @@
 # GSQ Vocabularies
-## Description
-The Geological Survey of Queensland's (GSQ's) vocabularies, formulated using SKOS, serialised as RDF (turtle) files.
+## Introduction
+The Geological Survey of Queensland (GSQ) publishes vocabularies - a way to describe things and the relationship between things.
```
Comment (KellyVance) [inline README.md]: New Line 3: Rewording is much better. But is "things" the best word? Subject? Entity? Concepts? Some combination?

Comment (dxwell) [reply README.md]: From https://www.w3.org/TR/owl-guide 
Every individual in the OWL world is a member of the class owl:Thing.
I think "thing" is ok.

File: README.md [hunk 2]
```diff
@@ -1,27 +1,64 @@
...
+
+## How to create a vocabulary
+<img src="vocabulary-build-pull-workflow.png" style="width:100%;" alt="Vocabulary build and pull process" />  
+
+**Fig. 2: Vocabulary build and pull process**
+
+1. Select the vocabulary editor of your choice.
+2. Create the vocabulary using the [SKOS Simple Knowledge Organization System](https://www.w3.org/TR/skos-reference/). See also the [SKOS Primer](https://www.w3.org/TR/skos-primer/) for the basics. NOTE: Always first check if there is an international or national vocabulary (see below for links).  
+    a. Use [Vocbench](www.vocbench.gsq.digital:7200/) to create the vocabulary.  
+    b. Use the Excel template to create the vocabulary. *(Vance can you upload the template to this repository and put the link here)*.
```
Comment (KellyVance) [inline README.md]: template added to TEST branch. new pull request needed

File: README.md [hunk 3]
```diff
@@ -1,27 +1,64 @@
...
+3. Export the vocabulary to a TTL file. If using Vocbench, it is easier to export the TTL from the Build repository in GraphDB. Follow the [instructions here](http://graphdb.ontotext.com/documentation/standard/exporting-data.html).
+4. Validate the TTL file using the [online Skosify tool](http://demo.seco.tkk.fi/skosify/skosify). Tick the checkboxes *Keep skos:related relationships within the same hierarchy* and *Include skos:narrower relations in output*
+5. Import the TTL file into a development branch in Github. Name your branch *dev-yourGithubusername*. See [how-to instructions here](https://guides.github.com/activities/hello-world/).
+6. When you're ready to publish your vocabulary into Test, submit a Pull Request to the TEST branch. See [how-to instructions here](https://guides.github.com/activities/hello-world/#pr).
+7. A member of the Data Integrity Team will review your vocabulary and either Approve or Request Changes. See [how-to instructions here](https://help.github.com/en/articles/approving-a-pull-request-with-required-reviews).
+8. If the pull request is approved, the vocab will now be in the TEST branch.
```
Comment (KellyVance) [inline README.md]: For approval can formalise apporval steps via pull requests? i.e. something like: creator(loads to)>dev>content authorities(peer review and approve pull request)>review>(Data managers approve data integrity and pull request)>test>(Data masters approve final publish to vocprez)>master

**Pull request closed without merging**